docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,528 @@
+# Notification-ex
+
+
+An application can use notifications-ex to keep the user informed of important information.
+Notification-ex api is designed to form more freely than existing notifications.
+
+The main features of the Notification-ex API include:
+
+- Creating a notification-ex items
+
+  You can [create a notification item](#create). There are 12 notification-ex item types and custom type.
+  There are [5 receiver-groups](#receiver) that you want your notification to be displayed on.
+
+- Creating a Group Item and add child
+
+  You can [create a group item](#group). You can include multiple items in a group in order.
+  You can create multiple groups and you can include it in another group.
+
+- Setting notification attributes
+
+  You can [set notification attributes](#set), such as the action, style, receiver group, main type etc.
+
+- Creating an Expanded Notification
+
+  You can [make a more extended form](#expanded) of notification.
+
+- Creating a Messaging style message
+
+  You can [make the messaging style](#messaging) notification.
+
+- Make a Notification Reporter Handle
+
+  You need to [create a reporter handle](#reporter) to post a note.
+  you can set to receive callbacks for events and errors.
+
+- Posting a notification
+
+  You can [post a single notification and the multiple notifications](#post) by using the `noti_ex_reporter_post()` function.
+
+- Updating notification content
+
+  You can [update the content of a notification](#update) using `noti_ex_reporter_update()` function.
+
+- Deleting a notification
+
+  You can [delete notifications](#delete) when no longer needed.
+
+## Prerequisites
+
+To enable your application to use the notification-ex functionality:
+
+1. To use the Notification-ex API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/notification</privilege>
+      <!--To launch an application by app_control handler-->
+      <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+   </privileges>
+   ```
+
+2. To use the functions and data types of the Notification-ex API, include the `<notification_ex.h>` header file in your application:
+   ```
+   #include <notification_ex.h>
+   ```
+
+   To ensure that a Notification function has been executed properly, make sure that the return is equal to `NOTI_EX_ERROR_NONE`.
+
+3. To follow this guide, place an image file in, for example, your application's shared resource directory. The following variables are used in the code:
+   ```
+   char *image_path[BUFLEN];
+   char *shared_path = app_get_shared_resource_path();
+   snprintf(image_path, BUFLEN, "%stutorial_native_api_application.png", shared_path);
+   free(shared_path);
+   ```
+
+<a name="create"></a>
```

Comment:
Please let me know how to use it.

Reply:
We have to navigate from a sentence or word to a heading. 
For example, 
We have a heading,
## Creating Expanded Notification
We have a sentence,
You can create [expanded](#creating-expanded-notification) notification.
Now, when we click "expanded", it will take us to the heading.

For more information, see https://github.com/Samsung/tizen-docs/blob/25f32714150ec711a1d287955ad5f5ea7964417c/styleguide/style.md#internal-links

Please let me know if you are not clear.